modules/ossm-rn-known-issues.adoc

```diff
@@ -36,6 +36,29 @@ endif::openshift-rosa[]
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-4056[OSSM-4056] Upgrading the `ServiceMeshControlPlane` from v2.2.x to v2.3.x or v2.4.x results in a TLS handshake failure after re-encryption on a terminated TLS gateway, `mode: SIMPLE`. The Ingress gateway returns a 503 error with the following message:
++
+`upstream connect error or disconnect/reset before headers. reset reason: connection termination`
++
+This error occurrs because the Istio Ingress gateway overwrites the Application Layer Protocol Negotiation (ALPN) after the re-encryption of terminated TLS traffic between the gateway and the sidecar.
++
+Workaround: Modify the `PeerAuthentication` specification so that TLS is disabled for the application.
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: disable-istio-mtls-for-tls-app
+  namespace: default
+spec:
+  celector: 
+    matchLabels:
+      …
+  mtls:
+    mode: DISABLE
+----
+
```

Comment:
We are postponing this bug fix to 2.4.3, so please remove it.
```suggestion
```

Reply:
Removed this known issue based on discussion with Jacek.